### ISSUE TITLE
Generate Backup Codes

### ISSUE THREAD

**Issue by joeydc** [Ylianst/MeshCentral#4195]
**Describe the bug**
The menu for generating backup code is missing.
Same issue on (2) MeshCentral installation.

**Screenshots**
![image](https://user-images.githubusercontent.com/8237849/176595086-a779a56f-70ba-44fc-b819-0b128ab6c31c.png) 
![image](https://user-images.githubusercontent.com/8237849/176596413-71d20f78-49b2-4a22-b54a-3f5f48d07910.png)

**Server Software (please complete the following information):**
 - OS: Debian 11 and Arch
 - Virtualization: Both installation are KVM virtual machine.
 - Network: Nginx reverse proxy for the Arch installation.
 - Version: 1.0.43
 - Node: 18.1.0
 - Browser: Firefox & Chromium

**Additional context**

I'm also missing some options in the the "Server Actions" section.

![image](https://user-images.githubusercontent.com/8237849/176596013-7133f4ca-7421-47c3-84ad-daa7370ba393.png)

![image](https://user-images.githubusercontent.com/8237849/176595956-fb9f36ab-2cef-43d9-b755-8fe1e07d25c3.png)

**Comment by si458** [comment]
tried it here with a fresh install of 1.0.43, the doesnt appear to be away of generating backup codes,
so this will need to be a feature request
also i have the same 4 options in the server actions on a fresh install so i dont believe this is a bug

**Comment by joeydc** [comment]
@si458 thank you !
The MeshCentral installation on Debian 11 is a fresh install, from yesterday.. and it's the one missing 2 options out of 4.

**Comment by si458** [comment]
can you check your `config.json`? the backup and restore should be true or not listed at all
https://github.com/Ylianst/MeshCentral/blob/aa58afcc3a5d738177ab7a7b6d0228d72af82b85/meshcentral-config-schema.json#L441-L480
```
{
  "domains": {
    "myServer": {
      "Backup": true,
      "Restore": true
    }
  }
}
```

**Comment by joeydc** [comment]
<details><summary>Here's my config file:</summary><p>


{
  "settings": {
    "cert": "removed",
    "_WANonly": true,
    "_LANonly": true,
    "SessionKey": "removed",
    "Port": 443,
    "_AliasPort": 443,
    "RedirPort": 80,
    "AgentPong": 300,
    "_TlsOffload": "10.0.2.254",
    "_RedirAliasPort": 80,
    "MongoDb": "mongodb://127.0.0.1:27017/meshcentral",
    "AllowLoginToken": true,
    "_AllowFraming": true,
    "_WebRTC": false,
    "_ClickOnce": false,
    "PublicPushNotifications": true,
    "_manageAllDeviceGroups": [ "user//admin" ],
    "_UserAllowedIP" : "127.0.0.1,::1,192.168.0.100",
    "plugins": {
                 "enabled": true
    }
  },
  "domains": {
    "": {
      "_certUrl": "removed",
      "Title": "removed",
      "Title2": "Remote Server",
      "_TitlePicture": "title-sample.png",
      "_Minify": true,
      "mstsc": true,
      "ssh": true,
      "_NewAccounts": true,
      "UserNameIsEmail": true
    }
  },
  "letsencrypt": {
    "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
    "email": "removed",
    "names": "removed",
    "production": true
  }


</p></details>

**Comment by silversword411** [comment]
"backup codes"...what do you mean?

Are you talking one time use 2FA recover codes?

**Comment by joeydc** [comment]
> "backup codes"...what do you mean?
> 
> Are you talking one time use 2FA recover codes?

Yes.

**Comment by ubiquity60441** [comment]
This is how it should look:

-----

![image](https://user-images.githubusercontent.com/108750909/177371339-807aabe7-c0ab-4c84-9d2c-8b6803a0bb5d.png)

-----
I found that I can get to this option by editing html in the browser:

-----

![image](https://user-images.githubusercontent.com/108750909/177371789-cac0cd9e-53ea-4044-b3de-63648d9d768d.png)

-----

remove the `style="display: none;"` and press enter or change to `style=""`

**Comment by joeydc** [comment]
@ubiquity60441 Thank you sir !
At least I can generate backup codes for now.

**Comment by si458** [comment]
`QV('manageOtp', (serverinfo.lock2factor != true) && (authFactorCount > 0) && ((features2 & 0x40000) == 0));`
just trying to piece together what is doing

**Comment by Ylianst** [comment]
Oh wow!!! @si458 found and fixed it!!!! I am going to release v1.0.46 with this shortly.
